_doc/dns.hook

```diff
@@ -32,22 +32,29 @@ set -e
 
 get_apex() {
   local name="$1"
-  if [ -z "$name" ]; then
-    echo "$0: couldn't get apex for $name" >&2
+  if [[ -z "$name" ]]; then
+    echo "$0: couldn't get apex for empty name" >&2
     return 1
   fi
-  if [ "`dig +noall +answer SOA "${name}." |grep SOA|wc -l`" == "1" ]; then
+  # Look up SOA to get APEX and nameserver
+  APEX_NS=$(dig +noall +answer SOA "$name"|awk 'NR==1 && $4 == "SOA" {print $5}')
```

Comment:
Out of curiousity, why wasn't this working? And there should definitely be a trailing dot after `$name`